pkg/operator/ceph/object/objectstore.go

```diff
@@ -916,18 +916,37 @@ func adjustZoneDefaultPools(zone map[string]interface{}, spec cephv1.ObjectShare
 		"user_uid_pool":   ".meta.users.uid",
 		"otp_pool":        ".otp",
 		"notif_pool":      ".log.notif",
+		"topics_pool":     ".meta.topics",  // introduced in Ceph v19
+		"account_pool":    ".meta.account", // introduced in Ceph v19
+		"group_pool":      ".meta.group",   // introduced in Ceph v19
 	}
 	for pool, nsSuffix := range zonePoolNSSuffix {
 		// replace rgw internal index pools with namespaced metadata pool
 		namespacedPool := defaultMetaPool + nsSuffix
-		prev, err := setObjProperty(zone, namespacedPool, pool)
+		prev, err := setObjPropertyIfExists(zone, namespacedPool, pool)
```

Comment:
For the pools added here, I think `setObjPropertyIfExists()` makes sense, but for existing pools, should we use `setObjProperty()` so that Rook can error out if the property is missing?

Reply:
i think that we already have described behaviour with `setObjPropertyIfExists()`. it will return error if pool is missing.
Old pools should always be presented. Even it is missing in ceph, `radosgw-admin` will set default values to json anyway. If it will be missing in json then we will see warning in rook logs.